toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -51,6 +51,7 @@
         "openssh-server",
         "openssl",
         "mariner-release",
+        "mariner-repos",
```

Comment:
I believe mariner-repos-shared is the package that you would like to install here instead of the mariner-repos package since this will also pull in the official repo file that's not needed.